### PR TITLE
fix(undo/redo): textarea undo/redo; italics, highlight, underline, strikethrough #717

### PR DESCRIPTION
**Pull request by arkRedM**

### CODE REVIEW

**Comment by tangjeff0** [comment]
Woweee

**Comment by arkRedM** [comment]
Screen Shot 2021-03-12 at 2 37 37 AM

Basic formatting and cmd z behavior are addressed in this patch. 
This is not related to undo/redo a txn at all. Maybe move the bounty to another issue?

**Comment by tangjeff0** [comment]
@arkRedM oh you're that's a separate issue, however, undo/redo should be implemented as a tx log. The current approach holds onto the entire last 10 db values, and drops and resets on undo/redo. This is extremely slow for big dbs - takes ~2 seconds for mine at the point.

I also think it's the basis for RTC, in terms of using a tx log.

**Comment by arkRedM** [comment]
> I also think it's the basis for RTC, in terms of using a tx log.

That's the reason I started with this issue as well 😆, quickly discovered that's unrelated. But I was already knee-deep, the only way out was to leave the patch is, in an ideal state -- hence added a few qol features while also fixing the root issue

File: src/cljs/athens/listeners.cljs [hunk 1]
```diff
@@ -99,7 +99,7 @@
                                        KeyCodes.BACKSLASH (if shift
                                                             (dispatch [:right-sidebar/toggle])
                                                             (dispatch [:left-sidebar/toggle]))
-                                       KeyCodes.H (util/toggle-10x)
+                                       KeyCodes.T (util/toggle-10x)
```
Comment (tangjeff0) [inline src/cljs/athens/listeners.cljs:102]: Great idea to use ctrl-T instead of ctrl-H. H always led to weird behavior.

**Review by tangjeff0 — APPROVED** [review]
CC #649 @dauberd @avidrucker

**Pull request merged**